pkg/operator/api/api.go

```diff
@@ -33,18 +33,20 @@ const (
 )
 
 type Cluster struct {
-	clientset kubernetes.Interface
-	Namespace string
-	Version   string
-	Replicas  int32
+	clientset   kubernetes.Interface
+	ClusterName string
```

Comment:
Nit: i would just name this variable `Name`. This already implies it is referring to the cluster name since the name of the struct is `Cluster`